great_ape_safe/great_ape_safe.py

```diff
@@ -45,6 +46,7 @@ def init_all(self):
         self.init_convex()
         self.init_curve()
         self.init_opolis()
+        self.init_univ3()
```

Comment:
can i suggest `init_uni_v3` here, i just made the same comment in jalbrekt's pr.

i think it is important to be consistent with this, so always snake case `protocol_something_v5`, unless it is a class; `ProtocolSomethingV5`. this is common python style.

(i would even say the same for `self.NonfungiblePositionManager`; this is adopting solidity style in python. imo should be `self.nonfungible_position_manager` or `self.nfp_manager`)

Reply:
noted, added your suggestion in last commit